src/domain/chainContext.ts

```diff
@@ -24,6 +24,30 @@ const WATCHDOG_FREQUENCY = 5 * 1000; // 5 seconds
 
 const MULTICALL3 = "0xcA11bde05977b3631167028862bE2a173976CA11";
 
+enum ChainSync {
+  SYNCED = "SYNCED",
+  SYNCING = "SYNCING",
```

Comment:
how do native people pronounce this? cause i always pronounce it like "Sinking" 🚢💦

Reply:
It indeed is pronounced like "sinking". English is indeed silly. :sweat_smile: